sympy/physics/control/lti.py

```diff
@@ -1300,6 +1300,56 @@ def to_expr(self):
             return Pow(self.den, -1, evaluate=False)
 
 
+class PIDController:
```

Comment:
I do realize this late but why is it an independent class?
If all the behavior matches that of a transfer function, should it not be a subclass?

Reply:
sir i think that making it a subclass of `Transferfunction()` will not be fruitful because of the inheritance issues....when i am making it a subclass of `Transferfunction()` i am getting recursion error and according to me it means that it is breaking the functionality of `Transferfunction()` class